tools/SetupFlow/DevHome.SetupFlow.Common/Contracts/InstallPackageTaskArguments.cs

```diff
@@ -1,36 +1,36 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using System.Collections.Generic;
-
-namespace DevHome.SetupFlow.Common.Contracts;
-
-/// <summary>
-/// Class representing an install package task's arguments passed to the elevated process.
-/// </summary>
-/// <remarks>
-/// <code>ElevatedProcess.exe --package-id id --package-catalog catalog</code>
-/// </remarks>
-public class InstallPackageTaskArguments : ITaskArguments
-{
-    private const string PackageIdArg = "--package-id";
-    private const string PackageCatalogArg = "--package-catalog";
+﻿// Copyright (c) Microsoft Corporation.

```

Comment:
looks like other than the elevation addition most of this file is a line ending change? Still not sure how to prevent this in VS